TASVideos.WikiEngine/MakeBracketed.cs

```diff
@@ -201,12 +201,16 @@ private static IEnumerable<INode> MakeLinkOrImage(int charStart, int charEnd, st
 		var pp = text.Split('|');
 		if (pp.Length >= 2 && IsLink(pp[0]) && IsImage(pp[1]))
 		{
-			return new[] { MakeLink(charStart, charEnd, pp[0], MakeImage(charStart, charEnd, pp, 1)) };
+			return new[] { MakeLink(charStart, charEnd, pp[0], MakeImage(charStart, charEnd, pp, 1, out _)) };
```

Comment:
Don't examine `unusedParams` here since in this case, there's nothing reasonable to disambiguate to (for now, anyway).